src/cmap/status.rs

```diff
@@ -15,11 +13,7 @@ impl Default for PoolStatus {
 
 /// Create a channel for publishing and receiving updates to the pool's generation.
 pub(super) fn channel() -> (PoolGenerationPublisher, PoolGenerationSubscriber) {
-    let (sender, mut receiver) = tokio::sync::watch::channel(Default::default());
```

Comment:
Since the `watch` channel no longer is seeded with an initial message, this is no longer necessary